tests/requirements/requirements_base.txt

```diff
@@ -9,13 +8,8 @@ django-app-helper
 factory-boy
 flake8
 isort
+ruff
```

Comment:
Should probably keep tooling changes as a separate PR, it's in now so no point wasting time reversing it. I would expect a pyproject.toml file replacing the setup.py file with other config changes for ruff to work correctly.